app/assets/javascripts/dul-argon-skin.js

```diff
@@ -26,6 +26,35 @@ $(document).ready(function() {
   /* enable tooltips*/
   $(function () {
     $('[data-toggle="tooltip"]').tooltip()
-  })
+  });
+
+});
+
+
+Blacklight.onLoad(function() {
+
+  	$(window).load(function(){
+
+      $('.document').each(function() {
+
+        var hasRequest = $(this).has("div.index-document-functions").length;
+        var hasWidth = $(this).children("div").find('img').get(0).naturalWidth;
+        console.log('width: ' + hasWidth);
```

Comment:
I'd remove this console log...

Reply:
Overall, looks great. Just that one logging change and it should be good to go.

Reply:
Thanks for the catch @seanaery - completely forgot I'd added that :(